lib/interim_wifi/if_supervisor.ex

```diff
@@ -36,6 +36,25 @@ defmodule Nerves.InterimWiFi.IFSupervisor do
     end
   end
 
+  def scan(ifname) do
+     pidname = pname(ifname)
+     if Process.whereis(pidname) do
+       GenServer.call(pidname, :scan, 30_000)
+     else
+       {:error, :not_started}
+     end
+  end
+
+  def status(ifname) do
+     pidname = pname(ifname)
+     if Process.whereis(pidname) do
+       # im not sure what status is needed here?
```

Comment:
@mobileoverlord Do you know what's the correct thing to have here? I'd like to remove this comment if this is the right thing to do here.